code/modules/power/fusion/fusion_reactions.dm

```diff
@@ -97,9 +97,6 @@
 	hidden_from_codex = TRUE
 
 /decl/fusion_reaction/phoron_supermatter/handle_reaction_special(var/obj/effect/fusion_em_field/holder)
-
-	wormhole_event(GetConnectedZlevels(holder))
```

Comment:
Not going to replace this?

Reply:
Gonna add it before merge

Reply:
Actually I don't know how to manually trigger events :D

Reply:
Does this actually start event? I see that in New setup() is called but unsure from there on